libraries/AP_BattMonitor/AP_BattMonitor_Params.cpp

```diff
@@ -20,29 +20,29 @@ const AP_Param::GroupInfo AP_BattMonitor_Params::var_info[] = {
 
     // @Param: VOLT_PIN
     // @DisplayName: Battery Voltage sensing pin
-    // @Description: Setting this to 0 ~ 13 will enable battery voltage sensing on pins A0 ~ A13. On the PX4-v1 it should be set to 100. On the Pixhawk, Pixracer and NAVIO boards it should be set to 2, Pixhawk2 Power2 is 13.
-    // @Values: -1:Disabled, 0:A0, 1:A1, 2:Pixhawk/Pixracer/Navio2/Pixhawk2_PM1, 13:Pixhawk2_PM2, 100:PX4-v1
+    // @Description: On the PX4-v1 it should be set to 100. On the Pixhawk, Pixracer and NAVIO boards it should be set to 2, Pixhawk2 Power2 is 13.
```

Comment:
The descrition doesn't need to lead in with v1. I suggest we don't even mention it. It's already listed in the values.

Reply:
Yeah probably worth a complete rewrite to actually use values. I was originally just deleting stale references with the minimum of rewording, but reading it standalone it's a terrible description. Will push an update a bit later.

Reply:
Pushed an update, I'm not thrilled with it though. Do you (or @hamishwillee) have any suggestions?

Reply:
Maybe "Sets the analog input pin that should be used for voltage monitoring."